src/game.ts

```diff
@@ -676,14 +673,10 @@ export default class Game {
 		this.resizeCombatFrame(); // Resize while the game is starting
 		this.UI.resizeDash();
 
-		const resizeGame = function () {
-			// @ts-expect-error 2683
+		const resizeGame = function (this: Game) {
```

Comment:
The usual way to do `.bind(this)` these days is to use an arrow function. Unlike other ways to define a function, they don't get their own `this`, so they're maybe less confusing.

People get confused with `bind`, probably in large part because it's one spot in JavaScript that's *really* different than other languages.

[More on arrow functions, if that happens to be helpful.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Keep it up!